admin_guide/service_accounts.adoc

```diff
@@ -64,8 +64,28 @@ To allow all service accounts in the *managers* project to edit resources in the
 ----
 $ oc policy add-role-to-group edit system:serviceaccounts:managers -n top-secret
 ----
-// end::sa-user-names-and-groups[]
 
+[[dedicated-admin-granting-service-accounts-access]]
+ifdef::openshift-dedicated[]
+== Granting Service Accounts Access to Dedicated-Admin Roles
+
+As an {product-title} administrator, you can now use service accounts to perform
+any actions that require {product-title} *admin* roles.
+
+The {product-title} *admin* service creates the *dedicated-admins group*. This
```

Comment:
Maybe place a link here to what the dedicated-admins group is? 
https://docs.openshift.com/dedicated/admin_guide/index.html#admin-guide-index-dedicated-admin-role

> the *dedicated-admins group*